integration-tests/model/test_simultaneous_SIQN.py

```diff
@@ -0,0 +1,262 @@
+"""
+This tests the use of simultaneous transport with the SIQN
+timestepping method. A few timesteps are taken with the Bryan-Fritsch
+bubble test case, which solves the Compressible Euler Equations.
+The two tracers of water vapour and cloud water are being tranpsported
+conservatively, which means they need to be transported simultaneously
+with the density.
+Degree 0 and 1 configurations are  tested to ensure that the simultaneous
+transport is working with the different wrappers.
+"""
+
+from os.path import join, abspath, dirname
+from firedrake import (
+    PeriodicIntervalMesh, ExtrudedMesh, SpatialCoordinate, conditional, cos, pi,
+    sqrt, NonlinearVariationalProblem, NonlinearVariationalSolver, TestFunction,
+    dx, TrialFunction, Function, as_vector, LinearVariationalProblem,
+    LinearVariationalSolver, Constant, BrokenElement
+)
+from gusto import *
+import pytest
+
+
+def run_simult_SIQN(tmpdir, order):
+
+    ncolumns = 50
```

Comment:
Can we reduce the resolution, since this is just for testing. I think 10 columns/layers should be enough for the higher-order case, and 20 for the lower-order case